frontend/src/components/MainFooter.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <footer class="main-footer">
+    <div class="container">
+      <div class="logo-title">
+        <a target="_blank" href="https://woowacourse.github.io/"
+          ><img class="logo" src="/assets/logo/logo_full_white.png"
+        /></a>
+        <h3 class="title"><router-link to="/recruits">지원</router-link></h3>
+      </div>
+      <div class="copyright">Copyright © 우아한테크코스. All rights reserved</div>
+    </div>
+  </footer>
+</template>
+
+<script>
+export default {}
+</script>
+
+<style scoped>
+.main-footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 30px 0;
+  background: #333;
+  color: #fff;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 800px;
+}
```

Comment:
드래그하여 블록설정을 방지하기 위하여 user-select: none을 넣어도 좋을 것 같아요~!
![image](https://user-images.githubusercontent.com/37579660/94873401-0a50d680-048a-11eb-92ff-7d0061292de1.png)

```suggestion
.container {
  display: flex;
  flex-direction: column;
  justify-content: center;
  width: 800px;
  user-select: none;
}
```

Reply:
오호.. 우형 채용사이트에도 해당 속성이 안걸려있네요😂
핀이 원하는대로 적용하면 될거 같아용